test/Sentry.Tests/ModuleInitializerAttribute.cs

```diff
@@ -1,4 +1,5 @@
-#if(!NET5_0)
+#if !NET5_0
```

Comment:
when .NET 6 comes around this will be a problem

Reply:
```suggestion
#if !NET5_0
```

Reply:
> when .NET 6 comes around this will be a problem

I know that happens to ma some times